requirements.txt

```diff
@@ -30,6 +30,7 @@ mysqlclient
 mock>=2.0.0
 mozilla-django-oidc==2.0.0
 passlib
+pymemcache==3.5.2
 pyopenssl
 pyquery
 pytest
```

Comment:
You still have py-memcached down at #L42 - do you need it after this change?x

Reply:
No. Removed in second commit